lib/Pusher.php

```diff
@@ -160,6 +160,9 @@ public function __construct($auth_key, $secret, $app_id, $options = array(), $ho
         // ensure host doesn't have a scheme prefix
         $this->settings['host'] =
             preg_replace('/http[s]?\:\/\//', '', $this->settings['host'], 1);
+
+        // curl handler
+        $this->ch = null;
```

Comment:
I think it should be defied as private variables and this code should be removed. 

This way it's going to be a private variable and could be accessed from anywhere.
thoughts?